### PR TITLE
*/etcd-member.yaml: update manifests to make it boot without CEO

### PR DESCRIPTION
**Pull request by alaypatel07**

### CODE REVIEW

File: templates/master/00-master/_base/files/etc-kubernetes-manifests-etcd-member.yaml [hunk 1]
```diff
@@ -47,28 +49,38 @@ contents:
...
           {{if .Images.clusterEtcdOperatorImageKey}}
-            COMMAND="cluster-etcd-operator mount"
+            SERVER_COMMAND="cluster-etcd-operator mount"
+            PEER_COMMAND="cluster-etcd-operator mount"
+            METRIC_COMMAND="cluster-etcd-operator mount"
           {{else}}
-            COMMAND="kube-client-agent request"
+            COMMAND="kube-client-agent request \
+                --dnsnames={{etcdServerCertDNSNames .}} \
+                --commonname=system:etcd-server:${ETCD_DNS_NAME} \"
```
Comment (hexfusion) [inline templates/master/00-master/_base/files/etc-kubernetes-manifests-etcd-member.yaml]: --commonname is a dupe here right?

Comment (alaypatel07) [reply templates/master/00-master/_base/files/etc-kubernetes-manifests-etcd-member.yaml]: nice catch, thanks

File: templates/master/00-master/_base/files/etc-kubernetes-manifests-etcd-member.yaml [hunk 2]
```diff
@@ -47,28 +49,38 @@ contents:
...
+            SERVER_COMMAND="$COMMAND
+                --ipaddrs=${ETCD_IPV4_ADDRESS},127.0.0.1 \"
```
Comment (hexfusion) [inline templates/master/00-master/_base/files/etc-kubernetes-manifests-etcd-member.yaml]: these need to add --dnsnames flags

Comment (hexfusion) [reply templates/master/00-master/_base/files/etc-kubernetes-manifests-etcd-member.yaml]: basically, if we are not using CEO rendered spec should look the same as it does today.

File: templates/master/00-master/_base/files/etc-kubernetes-manifests-etcd-member.yaml (hunk 1 above)
Comment (alaypatel07) [reply templates/master/00-master/_base/files/etc-kubernetes-manifests-etcd-member.yaml]: kubeconfig flag was missing, added, PTAL

**Review by hexfusion — APPROVED** [review]
/lgtm

**Pull request merged**